addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/internal/protocol/MilightDiscover.java

```diff
@@ -77,61 +86,59 @@ public MilightDiscover(DiscoverResult discoverResult, int resendTimeoutInMillis,
     }
 
     public void dispose() {
+        stopResend();
         willbeclosed = true;
         datagramSocket.close();
         try {
             join(500);
```

Comment:
I was not referring to stopResend, but thanks for adding the documentation there, it is clear. I was referring to the non-changed line which I added the comment on.